### DISCUSSION TITLE
How would i implement lua?

### DISCUSSION THREAD

**Discussion by axmtez** [CosmosOS/Cosmos#2799] [General]
**What is the usecase for this plug**
Using Lua With cosmos.

**Describe alternatives you've considered**
I tried using MoonSharp and NLua but they both give a IntPtr plug bug.

**Comment by zarlo** [comment]
you can fork the project and replace the code that needed to be pluged with some thing that does not have missing plugs or you can make the needed plugs yourself

or you can make your own lua run time

**Reply by axmtez** [reply]
wdym, how?

**Reply by zarlo** [reply]
you will need to find the parts of the .net std that they use that need plugs and make plugs for them or edit the code so it make use of other methods

**Reply by axmtez** [reply]
Are there any working libraries yet?
